pkg/northbound/gnmi/set.go

```diff
@@ -24,10 +24,10 @@ import (
 	"github.com/onosproject/onos-config/pkg/modelregistry/jsonvalues"
 	"github.com/onosproject/onos-config/pkg/store"
 	"github.com/onosproject/onos-config/pkg/store/change"
-	types "github.com/onosproject/onos-config/pkg/types/change/device"
+	devicechangetypes "github.com/onosproject/onos-config/pkg/types/change/device"
```

Comment:
maybe give it alias `devicetypes` - I've seen Jordan give the network equivalent the alias `networktypes`

Reply:
I wanted to clearly intend this was not the device types or devicepb that's the reason.